pkg/metrics/metrics.go

```diff
@@ -115,6 +115,7 @@ var (
 	OpsCount        *prometheus.CounterVec
 	OpsFailedCount  *prometheus.CounterVec
 	OpsSuccessCount *prometheus.CounterVec
+	OpsErrorCount   *prometheus.CounterVec
```

Comment:
Where is this initialized and registered ?

Reply:
ah looks like it got removed during rebase conflict handling.